Description: Modifies the detection of the JDK to build with any JDK
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/build/build.xml
+++ b/build/build.xml
@@ -49,12 +49,7 @@
     </not>
   </condition>
   <condition property="java5">
-    <or>
-      <equals arg1="${ant.java.version}" arg2="1.5"/>
-      <equals arg1="${ant.java.version}" arg2="1.6"/>
-      <equals arg1="${ant.java.version}" arg2="1.7"/>
-      <equals arg1="${ant.java.version}" arg2="1.8"/>
-    </or>
+    <istrue value="true"/>
   </condition>
 
   <!-- Common classpaths -->
